src/assets/app.extensions.json

```diff
@@ -560,7 +560,7 @@
               "click": "DELETE_LIBRARY"
             },
             "rules": {
-              "visible": "app.selection.library"
+              "visible": "app.toolbar.favorite.canEditMetadata"
```

Comment:
maybe we should refactor the rule id to be a little bit more generic since we reuse it here